tests/disaster-recovery/sc_arbiter/test_device_replacement.py

```diff
@@ -0,0 +1,64 @@
+# import random
+import logging
+
+# from ocs_ci.ocs import constants
```

Comment:
is this some draft PR? I see alot of code is commented

Reply:
yes, i need to make this wip. its because it is stuck